clients/venice-admin-tool/src/main/java/com/linkedin/venice/datarecovery/StoreRepushCommand.java

```diff
@@ -88,14 +114,106 @@ public List<String> getShellCmd() {
   }
 
   private void processOutput(String output, int exitCode) {
-    result.setStdOut(output);
-    result.setExitCode(exitCode);
-    result.parseStandardOutput();
-    result.setCoreWorkDone(true);
+    getResult().setStdOut(output);
+    getResult().setExitCode(exitCode);
+    getResult().parseStandardOutput();
+    getResult().setCoreWorkDone(true);
+  }
+
+  public ControllerClient buildControllerClient(String clusterName, String url, Optional<SSLFactory> sslFactory) {
+    return new ControllerClient(clusterName, url, sslFactory);
+  }
```

Comment:
Could we move this to the base class and remove the definitions in subclasses, as now all three subclasses need it.